float64[12] deg
